bears/r/FormatRBear.py

```diff
@@ -78,10 +78,13 @@ def create_arguments(filename, file, config_file,
         """
         options = {'source="' + escape(filename, '"\\') + '"',
                    'blank=' + _map_to_r_bool(r_keep_blank_lines),
-                   'brace.newline=' + _map_to_r_bool(r_braces_on_next_line),
                    'comment=' + _map_to_r_bool(r_keep_comments),
                    'arrow=' + _map_to_r_bool(r_use_arrows),
                    'indent=' + str(indent_size)}
+        if r_braces_on_next_line is not None:
```

Comment:
None leaves it as it is. For example if you have an if statement with brace next to it and another with if in the next line, the Format bear will make no changes when the default state is none. If True/false is specified either all braces move to next line or all braces move next to if.